src/constants/core.js

```diff
@@ -30,6 +31,8 @@ export const IDENTIFIERS = {
   EXTERNAL_SYSTEM_ID: 'EXTERNAL_SYSTEM_ID',
   INSTANCE_HRID: 'INSTANCE_HRID',
   ITEM_BARCODE: 'ITEM_BARCODE',
+  ISBN: 'ISBN',
+  ISSN: 'ISSN'
```

Comment:
Could you please add comma for consistency?

Reply:
Done! @Dmitriy-Litvinenko  please take a look again